interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/customize-appearance/report-visual-styles.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Report Visual Styles
+owner: Mary Sammal
+---
+# Report Visual Styles
+
+This topic describes how to combine [appearance properties](appearance-properties.md) into styles and apply them to report elements.
+
+## Create a Report Style
+
+Use the following approaches to create a visual style in your report:
+
+- **Extract a Style**
+
+    Specify [appearance properties](appearance-properties.md) for a report control and press **Extract Style** in the report's [toolbar](..\report-designer-tools\toolbar.md).
+
+    ![eurd-win-specify-properties](../../../../images/eurd-win-specify-properties.png)
+
+
+- **Create a new Style**
+
+    Press the caption button on the toolbar's Styles group to invoke the Styles Editor.
+
+    ![eurd-win-styles-add-styles](../../../../images/eurd-win-styles-add-style.png)
+
+    Press the **Add a style** button and specify appearance properties for the newly created style. Close the Styles Editor.
+
+The created style is added to the Styles gallery. You can access this gallery in the following places:
+
+- the Styles group in the report's [toolbar](..\report-designer-tools\toolbar.md);
+
+    ![eurd-win-styles-in-toolbar](../../../../images/eurd-win-styles-in-toolbar.png)
+
+- the Styles group in the [Report Explorer](..\report-designer-tools\ui-panels\report-explorer.md).
+
+    ![eurd-win-styles-in-report-explorer](../../../../images/eurd-win-styles-in-report-explorer.png)
+
+## Assign a Style to a Report Element
+
+Select a report element and press a style in the toolbar's Styles group.
+
+![eurd-win-assign-style](../../../../images/eurd-win-assign-style.png)
+
+
+This assigns the style to the report element's **Style** property.
+
+![eurd-win-style-property](../../../../images/eurd-win-style-property.png)
+
+
+## Assign Odd and Even Styles
+
+You can assign different styles to alternating rows in a report.
```

Comment:
Why are you writing it this way? Why 2 separate sentences?

You can assign different styles to alternating rows in a report.
->
You can use the **Odd Style** and **Even Style** properties to apply different styles to alternating rows in a report.

Reply:
I agree with you.

Reply:
Cool